tests/test_range.py

```diff
@@ -1205,6 +1204,89 @@ def test_fractional_duration(self):
                           datetime(2017, 6, 16, 19, 37, 19, 500000),
                           now)
 
+    def test_workday(self):
+        with freeze_time('2017-06-16 13:40:26'):  # Friday
+            self.assert_range('this workday', datetime(2017, 6, 16), datetime(2017, 6, 17))
+            self.assert_range('previous workday', datetime(2017, 6, 15), datetime(2017, 6, 16))
+            self.assert_range('previous 2 workdays', datetime(2017, 6, 14), datetime(2017, 6, 16))
+            self.assert_range('previous 3 workdays', datetime(2017, 6, 13), datetime(2017, 6, 16))
+            self.assert_range('previous 4 workdays', datetime(2017, 6, 12), datetime(2017, 6, 16))
+            self.assert_range('previous 5 workdays', datetime(2017, 6, 9), datetime(2017, 6, 16))
+            self.assert_range('previous 6 workdays', datetime(2017, 6, 8), datetime(2017, 6, 16))
+            self.assert_range('1 workdays ago', datetime(2017, 6, 15), datetime(2017, 6, 16))
+            self.assert_range('2 workdays ago', datetime(2017, 6, 14), datetime(2017, 6, 15))
+            self.assert_range('3 workdays ago', datetime(2017, 6, 13), datetime(2017, 6, 14))
+            self.assert_range('4 workdays ago', datetime(2017, 6, 12), datetime(2017, 6, 13))
+            self.assert_range('5 workdays ago', datetime(2017, 6, 9), datetime(2017, 6, 10))
+            self.assert_range('6 workdays ago', datetime(2017, 6, 8), datetime(2017, 6, 9))
+            self.assert_range('next workday', datetime(2017, 6, 19), datetime(2017, 6, 20))
+            self.assert_range('next 2 workdays', datetime(2017, 6, 19), datetime(2017, 6, 21))
+            self.assert_range('next 3 workdays', datetime(2017, 6, 19), datetime(2017, 6, 22))
+            self.assert_range('next 4 workdays', datetime(2017, 6, 19), datetime(2017, 6, 23))
+            self.assert_range('next 5 workdays', datetime(2017, 6, 19), datetime(2017, 6, 24))
+            self.assert_range('next 6 workdays', datetime(2017, 6, 19), datetime(2017, 6, 27))
```

Comment:
This range includes weekend days. This could be an issue if I say, "Give me the events from the next 6 workdays" but the range includes the weekend so I also could get events over the weekend as well.